src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -633,23 +636,10 @@ public function testProcessIsSignaledIfStopped()
         }
         $this->skipIfNotEnhancedSigchild();
 
-        $process = $this->getProcess(self::$phpBin.' -r "sleep(4);"');
+        $process = $this->getProcess(self::$phpBin.' -r "sleep(32);"');
         $process->start();
         $process->stop();
         $this->assertTrue($process->hasBeenSignaled());
-    }
-
-    public function testProcessWithTermSignal()
```

Comment:
adding the assertion to the previous test case is enough, no need for a specific test case here